BRB/PushButton.py

```diff
@@ -689,14 +657,14 @@ def GetResults(config, project, libraries):
                     reruncount += 1
                     outputDir, rv, sambaUpdate = globals()[pipeline](config, group, project, organism, libraryType, tuples)
```

Comment:
Here we are passing over the organism variable, which is the list of 3 strings. This is because the pipeline-calling functions (e.g. `scRNAseq()`) won't work with just one.